### PR TITLE
DRIVERS-2789 Fix Math Rendering in Client Side Encryption Spec

### PR DESCRIPTION
**Pull request by blink1073**
Please complete the following before merging:

- [ ] Update changelog.
- [ ] Make sure there are generated JSON files from the YAML test files.
- [ ] Test changes in at least one language driver.
- [ ] Test these changes against all server versions and topologies (including standalone, replica set, sharded
  clusters, and serverless).

### CODE REVIEW

**Review by W-A-James — CHANGES_REQUESTED** [review]
Running the following command, there are also instances of this pattern in `source/client-side-encryption/tests/README.md`. It does seem that these predate the conversion to markdown, but this seems like a good place to fix them.

```bash
grep -n '`\$.+\$`' **/*.md
```
Running the following should fix these instances and might be useful going forward as we continue to convert more specs to markdown

```bash
sed -i --regexp-extended 's/`(\$.+\$)`/\1/' **/*.md # Removes wrapping backticks in all markdown files and writes results in place
```

**Comment by blink1073** [comment]
@W-A-James what platform are you using?

When I run those commands I get no output from `grep`, and `sed: 1: "s/`(\$.+\$)`/\1/": \1 not defined in the RE` from the second command.

**Comment by W-A-James** [comment]
@blink1073 I'm running on Ubuntu 20.04 through WSL2.

**Comment by blink1073** [comment]
Thanks, I got `grep -E -n '`\$.+\$`' **/*.md` to work on MacOS, but I couldn't find an equivalent `sed` invocation that worked.

File: source/client-side-encryption/tests/README.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ The spec tests format is an extension of the
 - An `encrypted_fields` to set on the collection used for operations.
 - A `key_vault_data` of data that should be inserted in the key vault collection before each test.
 - Introduction `autoEncryptOpts` to `clientOptions`
-- Addition of `$db` to command in `command_started_event`
-- Addition of `$$type` to command_started_event and outcome.
+- Addition of $db`to command in`command_started_event\`
+- Addition of $$type\` to command_started_event and outcome.
```
Comment (W-A-James) [inline source/client-side-encryption/tests/README.md:36]:
```suggestion
- Addition of `$db`to command in`command_started_event`
- Addition of `$$type` to `command_started_event` and outcome.
```

File: source/client-side-encryption/tests/README.md [hunk 2]
```diff
@@ -32,10 +32,10 @@ The spec tests format is an extension of the
...
 
-The semantics of `$$type` is that any actual value matching one of the types indicated by either a BSON type string or
+The semantics of $$type\` is that any actual value matching one of the types indicated by either a BSON type string or
```
Comment (W-A-James) [inline source/client-side-encryption/tests/README.md:38]:
```suggestion
The semantics of `$$type` is that any actual value matching one of the types indicated by either a BSON type string or
```

File: source/client-side-encryption/tests/README.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ The following matches a command_started_event for an insert of a document where
     command_name: insert
 ```
 
-The values of `$$type` correspond to
+The values of $$type\` correspond to
```
Comment (W-A-James) [inline source/client-side-encryption/tests/README.md:67]:
```suggestion
The values of `$$type` correspond to
```

**Review by W-A-James — CHANGES_REQUESTED** [review]
Almost done, just a couple unmatched backticks

**Review by W-A-James — APPROVED** [review]

**Pull request merged**